src/assets/2D_Icons/Wave_person.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="50" height="50" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="25" cy="25.0001" r="25" fill="white"/>
+<path d="M25 17C26.6569 17 28 15.6569 28 14C28 12.3431 26.6569 11 25 11C23.3431 11 22 12.3431 22 14C22 15.6569 23.3431 17 25 17Z" fill="black"/>
```

Comment:
remove fill colors from these, they should be filled by theme

Reply:
done